doc/tutorial.md

```diff
@@ -0,0 +1,212 @@
+# Tutorial
+
+## Introduction
+This tutorial shows the steps for creating and instantiating a Dlite entity and populating it with data. For this, we will use an example `csv` file containing data corresponding to time measurements from a solar panel. The example file has the following format:
+
+
+| time                 | Impp [A] | Isc [A] | MPP [W] | Vmpp [V] | Voc [V] |
+|----------------------|----------|---------|---------|----------|---------|
+| 2022-08-01 00:41:00  |  1.188   |   1.25  |  38.182 |   32.14  |  37.33  |
+|  ...                 |   ...    |   ...   |   ...   |    ...   |   ...   |
+
+
+
+## Writing a DLite Entity
+Let us write an Entity representing our solar panel measurement. We start by creating a `json` file called `myEntity.json`.
+
+### **Step 1**: Giving it a unique identifier
+Firstly, we must provide a unique identifier for our Entity. There are several ways of doing this:
+
+#### **1.  Uniform Resource Identifier (URI)**
+We can create and assign a unique Uniform Resource Identifier (URI). A URI has the form **namespace/version/nameOfEntity**, and it can be either written directly in the json file:
+
+```json
+"uri": "http://www.ontotrans.eu/0.1/solarPanelMeasurement",
+```
+or we can give the **namespace**, **version**, and **name** of our Entity as separate fields:
+
+```json
+"name": "solarPanelMeasurement",
+"namespace": "http://www.ontotrans.eu/",
+"version": "0.1"
+```
+
+#### **2. Universally Unique Identifier (UUID)**
+Alternatively, we can generate and assign a universally unique identifier (UUID). In Python, this can be achieved by the use of the [UUID module](https://docs.python.org/3/library/uuid.html):
+
+```python
+import uuid
+
+# Generate a random UUID
+entity_uuid = uuid.uuid4()
+
+with open("myEntity.json", "w") as entity_file:
+    entity_file.write('"uuid": {}'.format(entity_uuid))
+```
+Our json file now contains the following:
+
+```json
+"uuid": <uuid>
+```
+where `<uuid>` is a unique identifier for our Entity. 
+
+
+### **Step 2**: Providing a link to the metadata
+Let us assume that we have followed option 1 and provided a URI. The next step consists in adding a **meta** field that will link the Entity to its metadata. In DLite, all Entities are instances of the *EntitySchema*. The *EntitySchema* defines the structure of the Entity. In the **meta** field of our Entity, we need to provide the URI of the EntitySchema. For a schematic overview of the DLite data structures and further detail, see [Concepts section of the DLite User Guide](https://sintef.github.io/dlite/user_guide/concepts.html). We add this information to our json file in the following manner:
+```json
+"uri": "http://www.ontotrans.eu/0.1/solarPanelMeasurement",
+"meta": "http://onto-ns.com/meta/0.3/EntitySchema",
```

Comment:
The `meta` field actually defaults to "http://onto-ns.com/meta/0.3/EntitySchema", making `meta` optional for entities. 

This was added as a convenience when creating entities.